terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1211,6 +1211,34 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                     mounts:
                     - ["/dev/nvme1n1", "/mnt/gitlab", "ext4", ""]
                     rancher:
+                      services:
+                        clamscan:
+                          image: clamav/clamav:0.104
```

Comment:
Extract a variable so the two mentions are coupled.

```suggestion
                          image: ${clamav_image}
```

Reply:
I think there is a typo in the suggestion, is `$` intentional?

Reply:
No, I confused the two interpolation mechanisms, f-string and TF reference.